tests/test_dom0_config.py

```diff
@@ -2,18 +2,18 @@
 import unittest
 
 STRETCH_TEMPLATES = [
-    "sd-svs-template",
-    "sd-svs-disp-template",
-    "sd-export-template",
+    "sd-app-template",
```

Comment:
how about 3. both? If the intent here is to ensure that old templates don't exist, we can just expand the list and call it OLD_TEMPLATES or something similar. because if we just do 2., the test won't fail for stretch users.

Reply:
yes I think old templates that contains STRETCH_TEMPLATES (prior to these changes) as well as the newly replaced templates (sd-svs-disp-buster-template, sd-svs-buster-template and sd-export-template) is a good idea

Reply:
Done in 550065c11a36182d935e86f9074980ffa026d4a4 (went with DEPRECATED_TEMPLATES)